### DISCUSSION TITLE
Can we get an msi installer

### DISCUSSION THREAD

**Discussion by schwit61** [valinet/ExplorerPatcher#3660] [Ideas]
Corporate policy blocks most .exe installers. msi installers rarely get blocked.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/discussions/2379

and you would also need the msi signed, signing is discussed here: https://github.com/valinet/ExplorerPatcher/discussions/396

**Comment by pyrates999** [comment]
and you are advised not to install this on work machines because it needs elevated privileges to install and update: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Comment by Amrsatrio** [comment]
Do you have admin access on your work PC? Meaning, can you put files in C:\Windows?

**Reply by schwit61** [reply]
Not full admin access. We use BeyondTrust which allows admin access on a case by case basis.

**Reply by Amrsatrio** [reply]
Yeah unfortunately you can't use EP for now. We have no plans for an MSI based installer. Rather, we are planning to:
1. Make a PowerShell-based script to download EP's files and put them in the designated places (of course requires admin). This way, the script's content will be transparent.
2. Copy explorer.exe into somewhere in your C:\Users\<Username> so that it can run with EP without injecting more stuff, but you won't be able to have the Windows 10 start menu. (Crashes with 11 taskbar w/o EP, crashes with EP for some reason, investigating this)

**Reply by schwit61** [reply]
Thanks. I was able to install OpenShell which is a major improvement. BTW, I had to extract the .msi from the openshell .exe installer to be able to do it.
